index.html

```diff
@@ -3,21 +3,34 @@
   <head>
     <title>gmail sender</title>
     <link rel="stylesheet" href="css/main.css" charset="utf-8">
+    <link rel="stylesheet" href="css/pikaday.css" charset="utf-8">
+    <script src="https://use.fontawesome.com/a46defc72c.js"></script>
+    <script src="js/pikaday.js"></script>
   </head>
   <body>
     <header>
-      <ul class='nav' role='nav'>
-        <li><a href="http://www.scheduled.email/auth/google_oauth2/new?redirect_to=http://localhost:8080">Sign In</a>
-        <li><a href="http://www.scheduled.email/sign-out?redirect_to=http://localhost:8080" class='new-window'>Sign Out</a>
-        <li><a href="#new-schedule" class="new-schedule">+ Schedule</a></li>
-        <li class = "logo"><h2 class = "headline">GMail Sender</h2></li>
-      </ul>
+      <div class='nav' role='nav'>
+        <ul>
+          <li>
+            <a href="http://www.scheduled.email/sign-out?redirect_to=http://localhost:8080" class='new-window'>Sign Out</a>
+          </li>
+          <li>
+            <div class = "outbox"><a href id = "outbox">outbox</a></div>
```

Comment:
Typically attributes for html elements don't have spaces around the `=` sign. So `class = "outbox"` should be `class="outbox"` to keep it consistent.